custom_components/detailed_hello_world_push/hub.py

```diff
@@ -13,7 +15,7 @@ class Hub:
 
     manufacturer = "Demonstration Corp"
 
-    def __init__(self, hass, host):
+    def __init__(self, hass, host) -> None:
```

Comment:
Please import `HomeAssistant` too above.
```suggestion
    def __init__(self, hass: HomeAssistant, host: str) -> None:
```